gallery/how_to/work_with_microtvm/micro_tflite.py

```diff
@@ -123,12 +123,18 @@
 # directory into a buffer
 
 import os
+import json
+import tarfile
+import pathlib
+import tempfile
 import numpy as np
 
 import tvm
-from tvm.contrib.download import download_testdata
 from tvm import relay
+import tvm.contrib.utils
+from tvm.contrib.download import download_testdata
 
+use_physical_hw = True if os.getenv("TVM_MICRO_USE_HW") else False
```

Comment:
minor point: cleaner is `bool(os.getenv("TVM_MICRO_USE_HW"))`

Reply:
done!